src/expac.c

```diff
@@ -69,7 +69,7 @@ typedef const char *(*extractfn)(void*);
 static int is_valid_size_unit(char *u)
 {
   return u[0] != '\0' && u[1] == '\0' &&
-    memchr(SIZE_TOKENS, *u, strlen(SIZE_TOKENS)) != NULL;
+    memchr(SIZE_TOKENS, *u, sizeof(SIZE_TOKENS) - 1) != NULL;
```

Comment:
There's no problem with strlen here. Calling it on a string literal is amortized to a compile-time constant.

Reply:
I later changed this from a macro expanding to a string constant, to a static const string which could be referenced as an array; strlen wouldn't be amortised there